core/Parsing.Documents.fs

```diff
@@ -228,11 +228,17 @@ let mutable languages = [
     lang "XML" "xsl" ".xml|.xsl"
         html
     lang "YAML" "" ".yaml|.yml"
-        /// Also allow text paragraphs to be wrapped. Though wrapping the whole
-        /// file at once will mess it up.
+        // Also allow text paragraphs to be wrapped. Though wrapping the whole
+        // file at once will mess it up.
```

Comment:
Not sure why adding the Zig entry below causes this error but I was running into this error:

```
./core/Parsing.Documents.fs(231,9): (232,42) info FSHARP: XML comment is not placed on a valid language element. (code 3520)
```